src/utilities/buildASTSchema.js

```diff
@@ -72,6 +73,11 @@ import {
   GraphQLSchema,
 } from '../type/schema';
 
+export type TypeFieldResolverMap = ObjMap<
+  | ObjMap<GraphQLFieldResolver<mixed, mixed, mixed>> /* type and interface */
+  | ObjMap<any> /* enum */,
```

Comment:
I don't like this much because `any` for enum will eat resolvers for object types.
Should I wrap enum values to object with key one key `values` or `enum`?
It may works better for flow and TS checking but feels weird :grimacing: @IvanGoncharov what do you think?